### DISCUSSION TITLE
Will nunomaduro/larastan always be an alias to larastan/larastan?

### DISCUSSION THREAD

**Discussion by mad-briller** [larastan/larastan#1881] [General]
Is there an end-date when `nunomaduro/larastan` will stop receiving updates? according to packagist it is still being updated as it has 2.9.2

**Comment by canvural** [comment]
I do not do anything special to update that 🤷🏽 Packagist is automatically keeping it updated I guess.

**Reply by mad-briller** [reply]
ah this may be caused by github's [redirect on rename](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository) feature, so packagist can still see `nunomaduro/larastan` but it actually reads from `larastan/larastan`

it seems this redirect will only stop happening if `nunomaduro/larastan` is recreated, so it seems this may be a "permanent" alias
